boot-eeprom-rpi4: Update language bootloader vs EEPROM and release names (#3649)
raspberrypi/documentation: documentation/asciidoc/computers/raspberry-pi/boot-eeprom-rpi4.adoc

* Run `sudo raspi-config`
* Select `Advanced Options`
* Select `Bootloader Version`
* Select `Default` for factory default settings or `Latest` for the latest bootloader release.
* Reboot

=== Updating the bootloader configuration

> The `DEFAULT` version of the bootloader represents the latest factory default firmware image. It's update for critical bug fixes, hardware support and periodically after features have been tested in the `LATEST` release. 

Reviewer: ```suggestion
The `DEFAULT` version of the bootloader represents the latest factory default firmware image. It updates to provide critical bug fixes, hardware support and periodically after features have been tested in the `LATEST` release. 
```
